NadekoBot.Core/Modules/Utility/QuoteCommands.cs

```diff
@@ -325,6 +330,64 @@ public async Task QuoteDelete(int id)
                     await ctx.Channel.SendErrorAsync(response).ConfigureAwait(false);
             }
 
+            [NadekoCommand, Usage, Description, Aliases]
+            [RequireContext(ContextType.Guild)]
+            [UserPerm(GuildPerm.Administrator)]
+            public async Task QuoteDeleteAuthor(IGuildUser usr)
+            {
+                
+                using (var uow = _db.GetDbContext())
+                {
+                    uow.Quotes.RemoveAllByAuthor(ctx.Guild.Id, usr.Id);
+                    await uow.SaveChangesAsync();
+        
+                }
+                var text = $"Deleted all quotes from {Format.Bold(usr.Username.SanitizeAllMentions())}";
+                await ctx.Channel.SendConfirmAsync(Format.Bold(ctx.User.ToString()) + " " + text);
+            }
+
+            [NadekoCommand, Usage, Description, Aliases]
+            [RequireContext(ContextType.Guild)]
+            [UserPerm(GuildPerm.Administrator)]
+            public async Task QuoteDeleteLinks()
+            {
+                int badlinks = 0;
+                //get all links from database
+                IEnumerable<Quote> quotes;
+                using (var uow = _db.GetDbContext())
+                {
+                    quotes = uow.Quotes.SearchQuoteLinkTextAsync(ctx.Guild.Id);
+                    
+                
+                foreach(var q in quotes){
+                    // check if valid url
+                    Uri uriResult;
+                    bool result = Uri.TryCreate(q.Text, UriKind.Absolute, out uriResult) && (uriResult.Scheme == Uri.UriSchemeHttp || uriResult.Scheme == Uri.UriSchemeHttps);
+                    if (result){
+                        using (var _http = _httpFactory.CreateClient("QuotesClient")){
+                        //check if link is dead
+                        try{
+                            await _http.GetStringAsync(q.Text);
+                        }
+                        catch (HttpRequestException ex){
+                            uow.Quotes.Remove(q);
+                            await uow.SaveChangesAsync();
+                            badlinks = badlinks+1;
+
+                        }
+                    }
```

Comment:
One useful heuristic for code readability is to reduce indentation as much as possible. So instead of 
```
if (result) {
    // do stuff
```
you can do
```
if (!result) {
    return;
}
// do stuff
```